README.md

```diff
@@ -102,6 +108,42 @@ stats_df = calculate_statistics(score_df, selected_df)
 print(stats_df)
 ```
 
+## Text-based Selective
+This example shows how to use text-based selection. In this scenario, we would like to select a subset of articles that is most diverse in the text embedding space and covers a range of topics. 
+
+```python
+# Import Selective and TextWiser
+import pandas as pd
+from feature.selector import Selective, SelectionMethod
+from textwiser import TextWiser, Embedding, Transformation
+
+# Data with the text content of each article
+data = pd.DataFrame({"article_1": ["article text here"],
+                     "article_2": ["article text here"],
+                     "article_3": ["article text here"],
+                     "article_4": ["article text here"]})
+
+# Labels to denote 0/1 coverage metadata for each article 
+# across three labels, e.g., sports, international, entertainment    
```

Comment:
Here we are saying 3 labels but the example is changed to 4 labels. Can we please change this example so that the number of articles and labels are of different size, and not 4 vs. 4. This might unintentionally suggest a square matrix here. That's why I went with 4 articles and 3 labels. If the tests does not work that way, try 5 artciles, 4 labels ?

Reply:
That's a good point! I changed it with 5 articles, 4 labels.